base/PHP5Daemon.php

```diff
@@ -100,6 +106,7 @@ private function isPHPCGIProcess(int $pid): bool {
 
   protected function getEnvironmentVariables(): Map<string, string> {
     return Map {
+      'OSS_PERF_TARGET' => (string) $this->target,
```

Comment:
is this meant to be in here? What's it useful for? Should there be a corresponding change to the HHVM runner?